File: python/lattice_weather/usage.py
"""usage_ids.payload_hash = D("WEATHER-USAGE/1", {delta, subject, unit}) (§3.3)."""

from .canon import D


def usage_payload_hash(subject: str, delta: str, unit: str) -> str:
    return D("WEATHER-USAGE/1", {"delta": delta, "subject": subject, "unit": unit})

File: packages/core/src/usage.ts
import { D } from "./hash.js";

/** usage_ids.payload_hash = D("WEATHER-USAGE/1", {delta, subject, unit}) (§3.3). */
export function usagePayloadHash(subject: string, delta: string, unit: "usd_micro"): string {
  return D("WEATHER-USAGE/1", { delta, subject, unit });
}
